recipes/heinz/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.0" %}
+
+
+package:
+  name: heinz
+  version: {{ version }}
+
+source:
+  fn: conda-heinz-v2.0.tar.gz
+  url: https://gitlab.com/ibivu/galaxy/conda-heinz/-/archive/v2.0/conda-heinz-v2.0.tar.gz
+  sha256: b3f94a703552387c18bbf6d42096ac858046b56b23b5792763d7cc98f27eeeb1
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build:
+    - gcc # [not osx]
+    - llvm  ==3.3 # [osx]
```

Comment:
is this strict pinning really needed?

Reply:
I have tried `gcc` on MacOS, but it does not work. I check the MakeFile, where the commands are for clang `-stdlib=libstdc++`; I guessed llvm might work, but llvm 4.0 does not, then I considered an older version, but I have not tested yet. But I decided to skip MacOS for now.